pdr_backend/models/wrapped_token.py

```diff
@@ -0,0 +1,38 @@
+from pdr_backend.models.token import Token
+from pdr_backend.util.web3_config import Web3Config
+
+
+class WrappedToken:
```

Comment:
Do we need a new class for WrappedToken? It's just another ERC20 token, after all.

Instead of a new class, what if we just had a separate function that converts a native token to ERC20 (or vice versa) ?